fido2.go

```diff
@@ -1052,6 +1227,8 @@ func errFromCode(code C.int) error {
 		return ErrOperationDenied
 	case C.FIDO_ERR_KEEPALIVE_CANCEL:
 		return ErrKeepaliveCancel
+	case C.FIDO_ERR_INVALID_OPTION:
+		return ErrInvalidOption
```

Comment:
This is the one change that called my attention, but I think it's fine - we don't check for this specific error in our sources.